viz/charts/linkerd-viz/README.md

```diff
@@ -95,13 +95,13 @@ Kubernetes: `>=1.21.0-0`
 | defaultRegistry | string | `"cr.l5d.io/linkerd"` | Docker registry for all viz components |
 | defaultUID | int | `2103` | UID for all the viz components |
 | enablePSP | bool | `false` | NodeAffinity section, See the [K8S documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) for more information nodeAffinity: -- Create Roles and RoleBindings to associate this extension's ServiceAccounts to the control plane PSP resource. This requires that `enabledPSP` is set to true on the control plane install. Note PSP has been deprecated since k8s v1.21 |
-| enablePodAntiAffinity | bool | `false` | Enables Pod Anti Affinity logic to balance the placement of replicas across hosts and zones for High Availability. Enable this only when you have multiple replicas of components. |
+| enablePodAntiAffinity | bool | `false` |  |
```

Comment:
Think the text for `enablePodAntiAffinity` was deleted by mistake. We should maybe add it back in and re-generate the README here 😅

Reply:
Was this removed in another commit on main? I only changed 1 line in the values.yaml and ran the doc script

Reply:
Fixed